code/_onclick/hud/screen_objects.dm

```diff
@@ -265,11 +265,6 @@
 	screen_loc = "7,7 to 10,8"
 
 
-/obj/screen/storage/New(new_master, mapload)
```

Comment:
conflicts with your other pr/out of scope

Reply:
I can solve the conflicts.

Reply:
okay but this is potentially breaking functionality here.

Reply:
Not really, the master already gets defined in the only place that creates a new instance of this object. This was just defining it again.
Chose to remove the `New()` proc and leave the definition alone due the whole thing needing some improvement.